Plugins/Flow.Launcher.Plugin.ProcessKiller/Main.cs

```diff
@@ -114,19 +114,14 @@ private List<Result> CreateResultsFromQuery(Query query)
                         {
                             processHelper.TryKill(p.Process);
                         }
-                        _ = DelayAndReQueryAsync(query.RawQuery); // Re-query after killing process to refresh process list
+                        // Re-query to refresh process list
+                        _context.API.ChangeQuery(query.RawQuery, true);
                         return true;
                     }
                 });
             }
 
             return sortedResults;
         }
-
-        private static async Task DelayAndReQueryAsync(string query)
-        {
-            await Task.Delay(500);
```

Comment:
Out of curiosity why the delay before re-query?

Reply:
It was to wait for process exit. Didn't realize I could do it the way in this PR.